src/pages/LoginPage/LoginPage.tsx

```diff
@@ -23,8 +23,8 @@ const LoginPage = () => {
         }}
       >
         <div className="login-form-header">
-          <h2 className="form-header">Welcome back!</h2>
-          <small className="form-header-description">
+          <h2 className="form-header text-preset-3">Welcome back!</h2>
+          <small className="form-header-description text-preset-6--regular">
```

Comment:
I don't think `small` is a good choice semantically. I'd use `p` or `span` instead